packages/sku/context/defaultCompilePackages.js

```diff
@@ -9,9 +9,15 @@ let detectedCompilePackages = [];
 
 try {
   detectedCompilePackages = glob
-    .sync([`node_modules/@seek/*/package.json`], {
-      cwd: cwd(),
-    })
+    .sync(
+      [
+        'node_modules/@seek/*/package.json',
+        'node_modules/.pnpm/@seek*/node_modules/@seek/*/package.json',
+      ],
+      {
+        cwd: cwd(),
+      },
+    )
     .map((packagePath) => {
```

Comment:
I wouldn't expect there to be any dupes though, the paths from each glob should always be different right?

Reply:
The dedupe should be done at the end, after filtering and extracting just the package name. Here's what I get when I run it in the Crackle monorepo (replacing `@seek` with `@crackle`):

```js
[
  '@crackle/babel-plugin-remove-exports',
  '@crackle/cli',
  '@crackle/core',
  '@crackle/router',
  '@crackle/babel-plugin-remove-exports',
  '@crackle/cli',
  '@crackle/core',
  '@crackle/babel-plugin-remove-exports',
  '@crackle/core',
  '@crackle/router',
  '@crackle/router',
]
```

because it finds the `package.json`s` in both places:

```
node_modules/@crackle/babel-plugin-remove-exports/package.json 
node_modules/@crackle/cli/package.json 
node_modules/@crackle/core/package.json 
node_modules/@crackle/router/package.json 
node_modules/.pnpm/@crackle+babel-plugin-remove-exports@0.2.1/node_modules/@crackle/babel-plugin-remove-exports/package.json 
node_modules/.pnpm/@crackle+cli@0.11.3_typescript@5.1.3/node_modules/@crackle/cli/package.json 
node_modules/.pnpm/@crackle+cli@0.11.3_typescript@5.1.3/node_modules/@crackle/core/package.json 
node_modules/.pnpm/@crackle+core@0.23.2_typescript@5.1.3/node_modules/@crackle/babel-plugin-remove-exports/package.json 
node_modules/.pnpm/@crackle+core@0.23.2_typescript@5.1.3/node_modules/@crackle/core/package.json 
node_modules/.pnpm/@crackle+core@0.23.2_typescript@5.1.3/node_modules/@crackle/router/package.json 
node_modules/.pnpm/@crackle+router@0.3.0_react-dom@18.2.0/node_modules/@crackle/router/package.json
```

Reply:
https://github.com/seek-oss/sku/pull/840/commits/6daaf0421e5ded82384663658df2be41d065e918